third-party/arcanum-worlds/odyssey-of-the-dragonlords/aw-ootd-weapons.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<elements>
+	<info>
+		<author url="https://www.arcanumworlds.com/">Arcanum Worlds</author>
```

Comment:
`<author url="https://www.arcanumworlds.com/">Arcanum Worlds</author>`

these lines at the top can be left out, they are not used anymore